standard/structs.md

```diff
@@ -89,9 +92,12 @@ struct_member_declaration
     | constructor_declaration
     | static_constructor_declaration
     | type_declaration
+    | fixed_size_buffer_declaration   // unsafe code support
     ;
 ```
 
+*fixed_size_buffer_declaration* is defined in [§23.8.2](unsafe-code.md#2382-fixed-size-buffer-declarations).
```

Comment:
Another case of the "*x* is defined in §clause" pattern as opposed to "*x* (§clause) is only available in unsafe code[§23]" one.